tox.ini

```diff
@@ -21,20 +22,22 @@ deps =
     dbt020: dbt-core==0.20.2
     dbt021: dbt-core==0.21.0
     dbt100: dbt-core==1.0.0
+    dbt130: dbt-core>1.2,<1.4.0
```

Comment:
@barrywhart This is the only constraint that will match any `1.3.0` prereleases, because `>1.2` filters out any `1.2.x` version and `pip` sorts `1.3.0b1` _before_ `1.3.0`. So, there's no release that matches your suggested constraint at the moment:

```console
pip install --pre "dbt-core>=1.3,<1.4.0"                             
ERROR: Could not find a version that satisfies the requirement dbt-core<1.4.0,>=1.3 (from
versions: 0.13.0a1, 0.13.0a2, 0.13.0rc1, 0.13.0, 0.13.1a1, 0.13.1a2, 0.13.1, 0.14.0a1,
0.14.0a2, 0.14.0rc1, 0.14.0, 0.14.1a1, 0.14.1rc1, 0.14.1rc2, 0.14.1, 0.14.2, 0.14.3rc1,
0.14.3, 0.14.4, 0.15.0b1, 0.15.0b2, 0.15.0b3, 0.15.0rc1, 0.15.0rc2, 0.15.0, 0.15.1rc1,
0.15.1rc2, 0.15.1, 0.15.2, 0.15.3rc1, 0.15.3, 0.16.0b1, 0.16.0b2, 0.16.0b3, 0.16.0rc1,
0.16.0rc2, 0.16.0rc3, 0.16.0rc4, 0.16.0, 0.16.1rc1, 0.16.1, 0.17.0b1, 0.17.0b2, 0.17.0rc1,
0.17.0rc2, 0.17.0rc3, 0.17.0rc4, 0.17.0, 0.17.1rc1, 0.17.1rc2, 0.17.1rc3, 0.17.1rc4, 0.17.1, 0.17.2b1, 0.17.2rc1, 0.17.2, 0.18.0b1, 0.18.0b2, 0.18.0rc1, 0.18.0rc2, 0.18.0,
0.18.1b1, 0.18.1b2, 0.18.1b3, 0.18.1rc1, 0.18.1, 0.18.2rc1, 0.18.2, 0.19.0b1, 0.19.0rc1,
0.19.0rc2, 0.19.0rc3, 0.19.0, 0.19.1b2, 0.19.1rc1, 0.19.1rc2, 0.19.1, 0.19.2rc1,
0.19.2rc2, 0.19.2, 0.20.0b1, 0.20.0rc1, 0.20.0rc2, 0.20.0, 0.20.1rc1, 0.20.1, 0.20.2rc1,
0.20.2rc2, 0.20.2, 0.21.0b1, 0.21.0b2, 0.21.0rc1, 0.21.0rc2, 0.21.0, 0.21.1rc1, 0.21.1rc2,
0.21.1, 1.0.0b1, 1.0.0b2, 1.0.0rc1, 1.0.0rc2, 1.0.0rc3, 1.0.0, 1.0.1rc1, 1.0.1, 1.0.2rc1,
1.0.2, 1.0.3, 1.0.4, 1.0.5rc1, 1.0.5rc2, 1.0.5rc3, 1.0.5, 1.0.6rc1, 1.0.6, 1.0.7, 1.0.8,
1.1.0b1, 1.1.0rc1, 1.1.0rc2, 1.1.0rc3, 1.1.0, 1.1.1rc1, 1.1.1rc2, 1.1.1, 1.1.2rc1, 1.1.2,
1.2.0b1, 1.2.0rc1, 1.2.0rc2, 1.2.0, 1.3.0b1)
ERROR: No matching distribution found for dbt-core<1.4.0,>=1.3
```